src/pages/index.js

```diff
@@ -48,14 +60,56 @@ addFormValidator.enableValidation();
 
 /*FUNCTIONS*/
 
+function handleLikeIcon(card) {
+  if (!card.isLiked) {
+    api
+      .isLiked(card.id)
+      .then((res) => {
+        card.updateIsLiked(res.isLiked);
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+  } else {
+    api
+      .unliked(card.id)
+      .then((res) => {
+        card.updateIsLiked(res.isLiked);
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+  }
+}
+
+function handleDeleteClick(card) {
+  deleteCardPopup.open();
+  deleteCardPopup.setSubmitAction(() => {
+    api
```

Comment:
Needs correction: you should also notify the user about the loading process by changing the confirm button text

Reply:
Still needs correction